style.css

```diff
@@ -0,0 +1,48 @@
+* {
+  box-sizing: border-box;
+  margin: 0px;
+  padding: 0px;
+  border: 0px;
+}
+
+body {
+  background-color: #CCC;
+  font-family: Georgia, serif;
+  padding: 0px;
+  position: relative;
+  min-height: 100%;
+}
+
+section {
+  width: 80%;
+  margin: 20px auto 20px auto;
```

Comment:
this line can be collapsed down to margin: 20px auto.

when dealing with any CSS property that can values for 1-4 of the sides (margin, padding, border, etc) you can short-hand the css properties.
```
including one value sets all 4 sides to that value,
including two values sets top/bottom to the first value and right/left to the second,
including three values sets top to the first value, bottom to the second, and right/left to the third,
including four values sets each side to the value in the following order top --> right --> bottom --> left  -- TRBL (trouble)
```